app/models/compute_resource.rb

```diff
@@ -123,6 +124,10 @@ def to_label
     "#{name} (#{provider_friendly_name})"
   end
 
+  def connection_options
+    http_proxy ? {:proxy => http_proxy.full_url} : {}
```

Comment:
hrmm, i guess if we do this, we'd need to make sure this is implemented for all compute resources?